content/wiki/projects.toml

```diff
@@ -75,3 +75,52 @@ language = ["Javascript"]
 source = "https://github.com/Permille/Permille.io"
 license = "CC BY-NC-SA 4.0"
 rendering = "Raytracing"
+
+[[projects]]
+name = "Gnome Tower"
+url = "https://www.youtube.com/channel/UC6qe1IBLMBi0jqaec-Msxlw"
+state = "Active Dev."
+language = ["C++"]
+rendering = "Rasterization"
+
+[[projects]]
+name = "Voxelman"
+url = "https://github.com/MrSmith33/voxelman2"
+state = "Active Dev."
+language = ["D", "Vox using Vulkan"]
```

Comment:
"using Vulkan" is not part of the language, but rather the graphics API/rendering.